README.md

```diff
@@ -6,23 +6,47 @@
 [![Known Vulnerabilities](https://snyk.io/test/github/natterstefan/react-wordpress-plugin/badge.svg)](https://snyk.io/test/github/natterstefan/react-wordpress-plugin)
 [![GitHub license](https://img.shields.io/github/license/natterstefan/react-wordpress-plugin.svg)](https://github.com/natterstefan/react-wordpress-plugin/blob/master/LICENCE)
 
-React-WordPress-Plugin enables you to use React on your admin-page
-and on the public-page alongside a fully functionally WordPress plugin.
+React-WordPress-Plugin is a WordPress Plugin boilerplate with React on both the
+admin page and React component rendering on the public page alongside a fully
+functionally WordPress plugin.
+
+### Features
+
+This boilerplate plugin provides certain out-of-the-box features for both development
+and production already:
```

Comment:
personal taste, but "already" sounds a little bit weird here. probably it's because there is also "out-of-the-box" in the same sentence?